Cargo.toml

```diff
@@ -17,6 +17,10 @@ documentation = "https://docs.rs/elements/"
 
 [dependencies]
 bitcoin = "0.23"
+# While this dependency is included in bitcoin, we need this to use the macros.
+# We should probably try keep this one in sync with the bitcoin version,
+# to avoid requiring two version of bitcoin_hashes.
+bitcoin_hashes = "0.7.6"
```

Comment:
Looks like rust-bitcoin is using >0.7.3 ... I think this is okay. Technically it means that somebody could pin their version at 0.7.3 and cargo wouldn't complain, but this seems like a pretty unlikely scenario.

Reply:
Ah @stevenroose says elsewhere that we need this to access the macros. Can you update the comment to reflect this?

Reply:
The first line of the comment says we need this to access the macros :)

If someone pinned their version to 0.7.3, he wouldn't be able to build elements.